src/main/java/edu/greenblitz/robotName/subsystems/Limelight/ObjectDetectionLimelight.java

```diff
@@ -0,0 +1,40 @@
+package edu.greenblitz.robotName.subsystems.Limelight;
+
+import edu.greenblitz.robotName.VisionConstants;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.networktables.NetworkTableEntry;
+import edu.wpi.first.networktables.NetworkTableInstance;
+
+
+public class ObjectDetectionLimelight {
+	private NetworkTableEntry notePositionEntry;
+	private static ObjectDetectionLimelight instance;
+
+	public ObjectDetectionLimelight(){
+		String name = VisionConstants.OBJECT_DETECTION_LIMELIGHT_NAME;
+
+		notePositionEntry = NetworkTableInstance.getDefault().getTable(name).getEntry("");
+	}
+
+	public static ObjectDetectionLimelight getInstance() {
+		return instance;
+	}
+
+	public static void init(){
+		if (instance == null) {
+			instance = new ObjectDetectionLimelight();
+		}
+	}
+
+	public Pose2d getObjectPosition(){
```

Comment:
we can do better I think, we can get the confidence value of the detection, so le't add a threshold